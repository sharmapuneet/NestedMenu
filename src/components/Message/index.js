import React from 'react'
import Typography from '@material-ui/core/Typography'

const Message = () => {
  return (
    <Typography variant="h1">
      This menu item has not been enabled yet.
    </Typography>
  )
}

export default Message
