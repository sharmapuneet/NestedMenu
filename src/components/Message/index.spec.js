import React from 'react'
import { shallow, render } from 'enzyme'
import Message from './index'

describe('Message', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Message />)

    expect(wrapper).toMatchSnapshot();
  })

  it('should have text', () => {
    const wrapper = render(<Message />)

    expect(wrapper.text()).toBe('This menu item has not been enabled yet.')
  })
})
