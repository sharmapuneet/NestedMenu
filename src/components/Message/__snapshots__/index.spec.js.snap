// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message should render correctly 1`] = `
<WithStyles(ForwardRef(Typography))
  variant="h1"
>
  This menu item has not been enabled yet.
</WithStyles(ForwardRef(Typography))>
`;
